core/baseTradingMethod.js

```diff
@@ -196,15 +196,20 @@ Base.prototype.addIndicator = function(name, type, parameters) {
 
   // some indicators need a price stream, others need full candles
   this.indicators[name].input = Indicators[type].input;
-} 
+}
 
 Base.prototype.advice = function(newPosition) {
-  if(!newPosition)
-    return this.emit('soft advice');
+  // Possible values are long and short. Long will trigger a buy method
+  // while short will trigger a sell method
+  var advice = 'soft';
+  if(newPosition) {
+    advice = newPosition;
+  }
 
   this.emit('advice', {
-    recommandation: newPosition,
-    portfolio: 1
+    recommandation: advice,
+    portfolio: 1,
+    datetime: moment().format('YYYY-MM-DD HH:mm:ss')
```

Comment:
Would you mind broadcasting the moment object itself instead? This makes it easier for other consumers to use. Actually it would probably be even better to use the time of the candle that "caused" the advice. (I can also do this after pulling this in).